integration-tests/hibernate-reactive-panache-agroal/pom.xml

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-hibernate-reactive-panache-agroal</artifactId>
```

Comment:
Panache is really not necessary as this happens with just hibernate reactive, but I had to make sure because we use panache reactive in our project, this can be reduced to using hibernate reactive extension only